src/app/core/services/http/http-rundown.service.ts

```diff
@@ -29,6 +29,10 @@ export class HttpRundownService implements RundownService {
     return this.http.put<void>(`${RUNDOWN_URL}/${rundownId}/activate`, null).pipe(catchError(error => this.httpErrorService.catchError(error)))
   }
 
+  public rehearsal(rundownId: string): Observable<void> {
```

Comment:
This is a method name should we should use the verb `rehearse`

Reply:
Fixed